### PR TITLE
Add logger

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to #302

### CODE REVIEW

File: src/middleware/logger.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-};
+// import { MiddlewareFn } from 'type-graphql';
+//
+// import { MyContext } from '../../types/MyContext';
+//
+// export const logger: MiddlewareFn<MyContext> = async ({ args }, next) => {
+//   logger.debug('args: ', args);
+//
+//   return next();
+// };
```
Comment (CarlosQ96) [inline src/middleware/logger.ts]: If this file is not gonna be used, let's delete it.

File: src/delete-uniswap/sdk.ts [hunk 2]
```diff
@@ -52,8 +53,8 @@ export default class Sdk {
     const route = new Route([pair], token);
     const price = route.midPrice.toSignificant(6);
 
-    // console.log('inv', route.midPrice.invert().toSignificant(6)) // 0.00496756
-    // console.log('price', price) // 201.306
+    // logger.debug('inv', route.midPrice.invert().toSignificant(6)) // 0.00496756
+    // logger.debug('price', price) // 201.306
```
Comment (CarlosQ96) [inline src/delete-uniswap/sdk.ts:58]: I assume this comments serve as documentation? but they will break if we try to use them. I'll add the dependency.

File: src/config.ts [hunk 3]
```diff
@@ -91,7 +92,7 @@ class Config {
     envVars.forEach(envVar => {
       if (envFile[envVar]) {
         this[envVar] = envFile[envVar];
-        // console.log(`envVar ---> : ${this[envVar]}`)
+        // logger.debug(`envVar ---> : ${this[envVar]}`)
```
Comment (CarlosQ96) [inline src/config.ts:96]: I'll add the logger dependency.

File: src/services/syncGivPrices.ts [hunk 4]
```diff
@@ -8,7 +8,7 @@ const cronJobTime =
   process.env.REVIEW_OLD_GIV_PRICES_CRONJOB_EXPRESSION || '0 0 * * *';
 
 export const runUpdateHistoricGivPrices = () => {
-  console.log('runUpdateHistoricGivPrices() has been called');
+  logger.debug('runUpdateHistoricGivPrices() has been called');
```
Comment (CarlosQ96) [inline src/services/syncGivPrices.ts:12]: This line broke we missed the import. I'll add it.

File: src/server/components/FilterListedComponent.jsx [hunk 5]
```diff
@@ -5,7 +5,7 @@ import { FormGroup, filterStyles, Label } from '@admin-bro/design-system'
 
 const FilterListedComponent = props => {
   const { onChange, property, theme } = props;
-  console.log(props);
+  logger.debug(props);
```
Comment (CarlosQ96) [inline src/server/components/FilterListedComponent.jsx]: Removing this, I added it for debugging purposes but it's not required. This is a adminBro custom component in jsx.

**Review by CarlosQ96 — CHANGES_REQUESTED** [review]

**Review by CarlosQ96 — APPROVED** [review]
Checked every single file didn't miss the import.
I added the missing dependencies on some files so they don't break and removed the old logger. Ran the server and a few processes and they seem to work fine.
Great that we set a rotation for the logger I think it's fine. Good job!
We can merge tomorrow.
LGTM!!

File: src/delete-uniswap/sdk.ts (hunk 2 above)
Comment (mohammadranjbarz) [reply src/delete-uniswap/sdk.ts:58]: Thanks man

**Comment by mohammadranjbarz** [comment]
> Checked every single file didn't miss the import. I added the missing dependencies on some files so they don't break and removed the old logger. Ran the server and a few processes and they seem to work fine. Great that we set a rotation for the logger I think it's fine. Good job! We can merge tomorrow. LGTM!!

Thanks for your review and fixing errors

**Pull request merged**